__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -17,7 +17,11 @@ const savePrepReplacementExpectation = [
   'lastUpdatedBy',
   'createdAt',
   'expiredAt',
-  '{}'
+  '{}',
+  null,
+  null,
+  null,
+  null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `null` values in the array seems unnecessary. If these are placeholders for new fields, it would be better to use descriptive strings instead of `null`. This will make the code more readable and maintainable.
```diff
-  '{}',
-  null,
-  null,
-  null,
-  null
+  '{}',
+  'anon_user_info_color_code',
+  'anon_user_info_color_name',
+  'anon_user_info_emoji_code',
+  'anon_user_info_emoji_name'
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->